buildmod.R

```diff
@@ -0,0 +1,23 @@
+# Aim: build custom builds of the Propensity to Cycle Tool
+# This is a modified version of buildmaster.R
```

Comment:
Is there a reason to keep `buildmod.R` and `buildmaster.R`?  Duplication isn't ideal...